### PR TITLE
Update Minigo-prow config so that we can use bootstrap.py

### PR DESCRIPTION
**Pull request by Kashomon**
This updates the minigo-prow config to enable us to use bootstrap.py.

Corresponding minigo change: https://github.com/tensorflow/minigo/pull/79

### CODE REVIEW

File: jobs/config.json [hunk 1]
```diff
@@ -11358,5 +11358,14 @@
     "sigOwners": [
       "sig-big-data"
     ]
+  },
+  "minigo-testing": {
```
Comment (Kashomon) [inline jobs/config.json]: I feel like this needs to link into the config below somehow?

File: prow/config.yaml [hunk 2]
```diff
@@ -4934,8 +4934,14 @@ presubmits:
       preset-service-account: true
     spec:
       containers:
-      - image: gcr.io/minigo-testing/minigo-prow-harness:latest
+      - image: gcr.io/minigo-testing/minigo-prow-harness-v2:latest
```
Comment (krzyzacy) [inline prow/config.yaml]: can you link your Dockerfile?

File: jobs/config.json (hunk 1 above)
Comment (krzyzacy) [reply jobs/config.json]: is this yet another job?

File: prow/config.yaml (hunk 2 above)
Comment (Kashomon) [reply prow/config.yaml]: Not submitted yet: https://github.com/tensorflow/minigo/pull/79

File: jobs/config.json (hunk 1 above)
Comment (Kashomon) [reply jobs/config.json]: Ah, oops. This is me not understanding how this file works. Fixed now.

File: prow/config.yaml (hunk 2 above)
Comment (krzyzacy) [reply prow/config.yaml]: I guess... why not just use existing kubekins image? because of the pinned gcloud version?

Comment (Kashomon) [reply prow/config.yaml]: Oh, I dunno. I guess we could do that. But what about our other random deps that we need to install? Like Python 3.6

Comment (krzyzacy) [reply prow/config.yaml]: you can build your image base off the existing images and add your deps?

Comment (Kashomon) [reply prow/config.yaml]: I think that's true, but that shouldn't affect this config/pr. We'll still need our own prow-harness-image and then we'll install our deps.

Comment (BenTheElder) [reply prow/config.yaml]: Hahaha: 
https://github.com/kubernetes/test-infra/blob/a1a939885b73e8b180f49018ece25c3e3b9e94db/experiment/nursery/create_bazel_cache_rcs.sh#L51
You're not using bazel though right? :-)
Not a blocker, just amusing to immediately see someone violating this assumption.

File: prow/config.yaml [hunk 3]
```diff
@@ -14142,6 +14154,11 @@ periodics:
...
   spec:
     containers:
-    - image: gcr.io/minigo-testing/minigo-prow-harness:latest
+    - image: gcr.io/minigo-testing/minigo-prow-harness-v2:latest
       imagePullPolicy: Always
-
+      args:
+      - "--clean"
+      - "--job=$(JOB_NAME)"
+      - "--repo=tensorflow/minigo=$(PULL_REFS)"
```
Comment (krzyzacy) [inline prow/config.yaml]: periodic job does not have $PULL_REFS, just use master

File: prow/config.yaml (hunk 2 above)
Comment (krzyzacy) [reply prow/config.yaml]: 🤦‍♂️

**Comment by krzyzacy** [comment]
and please squash :-)
/this-is-fine
/meow sunglasses
/joke

File: prow/config.yaml (hunk 3 above)
Comment (Kashomon) [reply prow/config.yaml]: How's this?

File: prow/config.yaml (hunk 2 above)
Comment (Kashomon) [reply prow/config.yaml]: Ha!

**Comment by Kashomon** [comment]
Ok, should be all squashed now.

**Pull request merged**